arithmetization/src/main/java/net/consensys/linea/zktracer/runtime/stack/StackOperation.java

```diff
@@ -50,7 +50,11 @@ public final class StackOperation {
    */
   @Getter private final int stackStamp;
 
-  StackOperation() {
+  /** Singleton ``empty stack operation'' object. */
+  public static final StackOperation EmptyStackOperation = new StackOperation();
```

Comment:
Make EmptyStackOperation lower camel case, i.e : `emptyStackOperation`

Reply:
I agree, but I used to see  lower camel case when it is the instance of a singleton.

Reply:
I implemented the factory method (AFAICT) with a Lombok getter. Intellij renamed the empty stack item to `EMPTY_STACK_ITEM`. Also went renamed `StackOperation` to `StackItem` (in line with specification.)